pkgs/development/compilers/rust/binary.nix

```diff
@@ -52,6 +52,12 @@ rec {
       # https://github.com/rust-lang/rust/issues/34722#issuecomment-232164943
     '';
 
+    # The strip tool in cctools 973.0.1 and up appears to break rlibs in the
+    # binaries. The lib.rmeta object inside the ar archive should contain an
+    # .rmeta section, but it is removed. Luckily, this doesn't appear to be an
+    # issue for Rust builds produced by Nix.
```

Comment:
For future readers, this isn't an issue for the Rust builds produced after bootstrapping because we already disable stripping there:
https://github.com/NixOS/nixpkgs/blob/d3c2379d149073e53eb5580deafad475c1cabe28/pkgs/development/compilers/rust/rustc.nix#L35-42

Reply:
For extra posterity, the PR that added the line linked above: 
- https://github.com/NixOS/nixpkgs/pull/34227